drivers/spi/spi_ll_stm32.c

```diff
@@ -818,6 +818,24 @@ static const struct spi_driver_api api_funcs = {
 	.release = spi_stm32_release,
 };
 
+static int spi_stm32_cs_gpios_configure(const struct device *dev)
+{
+	int err;
+	const struct spi_cs_control *cs_gpio;
+	struct spi_stm32_data *data = dev->data;
+
+	for (cs_gpio = data->ctx.cs_gpios;
+	     cs_gpio < &data->ctx.cs_gpios[data->ctx.num_cs_gpios];
+	     cs_gpio++) {
+		err = spi_cs_configure(cs_gpio);
+		if (err) {
+			return err;
+		}
+	}
+
+	return 0;
+}
+
```

Comment:
There is nothing stm32 specific in this function, so I would rather have one generic function that already iterates through all CS gpios.

Reply:
I'm aware of that but I wasn't sure if I should create another function like e.g spi_cs_configure_all that will call spi_cs_configure or create the separate functions for each arch SPI driver and call them in the init function. Now I see that a generic approach would be more convenient so I'm gonna fix that.